netutils/config/compliance.py

```diff
@@ -20,9 +20,14 @@
     "aruba_aoscx": parser.ArubaConfigParser,
     "mrv_optiswitch": parser.OptiswitchConfigParser,
     "extreme_netiron": parser.NetironConfigParser,
+    "paloalto_panos": parser.PaloAltoNetworksConfigParser,
     "mikrotik_routeros": parser.RouterOSConfigParser,
 }
 
+conversion_map: t.Dict[str, t.Type[parser.BaseConfigParser]] = {
+    "paloalto_panos": parser.PaloAltoNetworksConfigParser,
+}
+
```

Comment:
This should be:

* Move to conversion.py
* Map to `'paloalto_panos_brace_to_set': paloalto_panos_brace_to_set`
* Point the test to ^^